designer/server/src/main/scala/pl/touk/nussknacker/ui/util/TapirCodecs.scala

```diff
@@ -0,0 +1,91 @@
+package pl.touk.nussknacker.ui.util
```

Comment:
The same as above - we need some package for things related with server logic handling